engine/src/main/java/org/terasology/engine/logic/behavior/actions/conditions/ConditionAction.java

```diff
@@ -98,7 +98,7 @@ protected boolean condition(Actor actor) throws ClassNotFoundException, NoSuchFi
                                 secondValue = "";
                                 break;
                             default:
-                                logger.error("Unsupported guard value type: {}", tokens[0]);
+                                logger.atError().addArgument(() -> tokens[0]).log("Unsupported guard value type: {}");
```

Comment:
The associated method is expected to only be evaluated if the default case applies.
In this case the log is required, so the evaluation is not undesired and the PMD warning should be ignored:
```suggestion
                                logger.error("Unsupported guard value type: {}", tokens[0]); //NOPMD
```